broadcast/internal/arc/arc_client.go

```diff
@@ -0,0 +1,35 @@
+package arc
+
+import (
+	broadcast_api "github.com/bitcoin-sv/go-broadcast-client/broadcast"
+	"github.com/bitcoin-sv/go-broadcast-client/broadcast/internal/httpclient"
+)
+
+const (
+	arcQueryTxRoute     = "/v1/tx/"
+	arcPolicyQuoteRoute = "/v1/policy"
+	arcSubmitTxRoute    = "/v1/tx"
+	arcSubmitTxsRoute   = "/v1/txs"
+)
+
+type Config interface {
```

Comment:
Reduncant interface, we can use `ArcClientConfig`

Reply:
@arkadiuszos4chain we have a problem removing this interface. It causes circullar dependency when we try to import broadcast_client.ArcClientConfig here.